runtime/src/lib.rs

```diff
@@ -2,19 +2,19 @@
 // `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
 #![recursion_limit = "256"]
 
-// Make the WASM binary available.
+extern crate alloc; // Make the WASM binary available.
```

Comment:
```suggestion
extern crate alloc;

// Make the WASM binary available.
```

It's confusing if the comment is in the same line.

Reply:
It isn't in my source locally, not sure what's going on there.

Reply:
Re-fetched and got it locally now, pushing a fix.